action.yml

```
@@ -10,11 +10,12 @@ inputs:
   terminus-version:
     description: |
       The full version of Terminus to install. If omitted, the latest version is used.
+      For deterministic builds, specifying a version is recommended.
```

```suggestion
      Specifying a version provides deterministic results in exchange for not always having the latest version, with the caveat that deprecated versions will eventually be rejected by the the back-end.
```

I like the idea of calling out specifying a version for deterministic results builds, but I don't think we want to make that a blanket recommendation for all users.  Terminus evolves over time and **we plan to begin hard-refusing requests from deprecated versions server-side in the near future**.  

Hard-coding a version would mean the customer would need to update their CI configuration from time to time.  For larger enterprise customers, that's a reasonable tradeoff, but it's not a good tradeoff for everyone.